chainer/functions/array/broadcast.py

```diff
@@ -154,4 +164,10 @@ def broadcast_to(x, shape):
                [0, 1, 2]])
 
     """
-    return BroadcastTo(shape)(x)
+    if x.shape == shape:
+        if isinstance(x, variable.Variable):
+            return x
+        else:
+            return variable.Variable(x, requires_grad=False)
```

Comment:
I feel this code fragment should be separated into a common utility function (like `chainer.to_variable`)...

Reply:
I fixed